Update player.cc

yrmvgh opened the pull request:
I am 90% certain these changes will work. By simplifying the lava / fire boost into just "lava boost", there's no longer a case where at high temp you get some of the benefits of lower temperature tiers, but not all.

If max temp orcs can't read books, also, that saves us some length in the text string. :D ("can't read")

yrmvgh committed:
Update player.cc

I am 90% certain these changes will work. By simplifying the lava / fire boost into just "lava boost", there's no longer a case where at high temp you get some of the benefits of lower temperature tiers, but not all.

If max temp orcs can't read books, also, that saves us some length in the text string. :D ("can't read")

yrmvgh committed:
Update player.cc

when you're cool, mv costs +100. I think that means that above that, you return to normal automatically. but maybe the room temp lines need to get uncommented? I think they'll make you above normal speed though

jeremygurr commented on crawl-ref/source/player.cc:
```diff
@@ -6458,14 +6461,17 @@ int player::racial_ac(bool temp) const
-            return 100 * experience_level / 3;              // max 9
+            return 100 * experience_level / 3;  		// max 9
+        if (species == SP_LAVA_ORC)
+            && temperature() < TEMP_WARM; 			// 1-8
```
this needs to be inside the parentheses or it won't compile

yrmvgh committed:
Update player.cc

Thanks @jeremygurr - do you have any guess whether this will cause your AC to be "highlighted" in another color to indicate it's buffed? I suspect it won't, but I'm hoping it will.

jeremygurr commented:
This one has conflicts. Could you resolve the conflicts or apply these changes to the latest code?

yrmvgh commented:
Damn. I will try. It says
"This branch has conflicts that must be resolved
Only those with write access to this repository can merge pull requests. "
and I think that means I can't "resolve" conflicts. I will put it into master again. Are you done with master for today?

jeremygurr commented:
yeah no major changes will happen any time soon
